src/components/Radio/index.tsx

```diff
@@ -0,0 +1,112 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import styled from '@emotion/styled'
+
+export interface RadioProps extends HTMLAttributes<HTMLDivElement> {
+  formName: string
+  items: { code: string; name: string }[]
+  radioDirection: string
+  onChange(e: React.FormEvent<HTMLInputElement>): void
+}
+
+interface FormProps extends HTMLAttributes<HTMLDivElement> {
+  radioDirection: string
+}
+const Radio = ({
+  formName,
+  onChange,
+  items,
+  radioDirection = 'horizental',
+  ...props
+}: RadioProps): ReactElement => {
+  const handleRadiobutton = (e: React.FormEvent<HTMLInputElement>): void => {
+    onChange && onChange(e)
+  }
+
+  const radioList = items?.map(({ code, name }) => (
+    <StyledInputWrapper key={code} className={`${radioDirection}`}>
+      {name}
+      <StyledInput
+        id={code}
+        name={formName}
+        type="radio"
+        value={code}
+        onChange={(e): void => {
+          handleRadiobutton(e)
+        }}
```

Comment:
```suggestion
        onChange={handleRadiobutton}
```
요러케 축약할 수 있을 것 같슴니다!